.setup/options/develop/web/php/web-server.sh

```diff
@@ -49,3 +49,6 @@ elif [ "$WEB_SERVER" = "nginx" ]; then
 else
     echo "Web server is not set"
 fi
+
+bash mariadb.sh
+bash php.sh "$WEB_SERVER"
```

Comment:
pass it as an argument when calling the script to make it available in the script